mcp_zammad/server.py

```
@@ -675,37 +990,117 @@
                 )
-                return Ticket(**ticket_data)
+                ticket = Ticket(**ticket_data)
+
+                # Format response based on preference
+                if params.response_format == ResponseFormat.JSON:
+                    result = json.dumps(ticket.model_dump(), indent=2, default=str)
+                else:  # MARKDOWN (default)
+                    result = _format_ticket_detail_markdown(ticket)
+
+                return truncate_response(result)
             except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)